.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
+  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+  
+  If you're publishing a new extension to OpenVSX, please make sure the extension has an OSI-approved open source license. If you want to have an extension with a proprietary or non-approved license, please ask its maintainers to publish it.
+-->
+<!--
+
+PLEASE READ THE FOLLOWING BEFORE SUBMITTING THIS PULL REQUEST:
+
+## A note about PRs adding or fixing extensions
+
+### For extension authors
+
+Although this repository is all about extensions, it is very hard to manage houndreds of extensions just by our team, we don't have the capacity to take care of every new broken extension, which includes new extensions. If you are the author of this contributed extension, please take the time to integrate OpenVSX publishing into your extension. You can refer to our [direct publish setup](docs/direct_publish_setup.md) doc for a guide on how to do it easily.
```

Comment:
I'd suggest we directly copy/paste the reasoning from the README for consistency, but also as it's more accurate, and also explains a bit more about _why_ it's better for authors to publish (and not just that it's a pain for us... which is true, that also!). But I'd lead with the benefits to the extension authors and/or the problems which occur when authors don't publish themselves. 

```suggestion
`publish-extensions` exists to seed the Open VSX marketplace, and also serves as a place for extensions that cannot feasibly be published directly by the extensions authors. In the long-run it is better for extension owners to publish their own plugins because:

1. Any future issues (features/bugs) with any published extensions in Open VSX will be directed to their original repo/source-control, and not confused with this repo publish-extensions.
2. Extensions published by official authors are shown within the Open VSX marketplace as such. Whereas extensions published via publish-extensions display a warning that the publisher (this repository) is not the official author.
3. Extension owners who publish their own extensions get greater flexibility on the publishing/release process, therefore ensure more accuracy/stability. For instance, in some cases publish-extensions has build steps within this repository, which can cause some uploaded plugin versions to break (e.g. if a plugin build step changes).

If you are the author of the extension being raised in this PR, please first consider directly publishing the extension yourself. You can refer to our [direct publish setup](docs/direct_publish_setup.md) doc for a guide on how to publish your plugin to Open VSX.
```